### PR TITLE
Add HF pointers

### PR DESCRIPTION
**Pull request by osanseviero**
Hi there :hugs: In collaboration with Google, we added the gemma flax files to the Hugging Face Hub. This PR adds instructions to manually or programmatically download the models.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,9 +34,21 @@ pip install git+https://github.com/google-deepmind/gemma.git
 ### Downloading the models
 
 The model checkpoints are available through Kaggle at
-http://kaggle.com/models/google/gemma. Select one of the **Flax** model
+https://www.kaggle.com/models/google/gemma/frameworks/flax. Select one of the **Flax** model
```
Comment (Molugan) [inline README.md:37]: Please rreverse back this line.

Comment (osanseviero) [reply README.md]: Done!

**Comment by Molugan** [comment]
Hello,

Sorry for the delay. For a security check, can you email me the name of your point of contact at Google so I can confirm with them before approving the PR ? I apologize for the mess.

**Comment by Molugan** [comment]
All clear

**Pull request merged**